kedro-datasets/test_requirements.txt

```diff
@@ -53,8 +53,8 @@ scikit-learn~=1.0.2
 scipy~=1.7.3
 snowflake-snowpark-python~=1.0.0; python_version == '3.8'
 SQLAlchemy>=1.4, <3.0  # The `Inspector.has_table()` method replaces the `Engine.has_table()` method in version 1.4.
-tables~=3.6.0; platform_system == "Windows" and python_version < '3.9'
-tables~=3.6; platform_system != "Windows"
+tables~=3.7.0; platform_system == "Windows"
+tables~=3.7; platform_system != "Windows"
```

Comment:
Can't these two lines just be merged to `tables~=3.7` now?

Reply:
my bad, yes!